presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -889,6 +918,10 @@ private boolean canQueueMore()
     {
         checkState(Thread.holdsLock(root), "Must hold lock");
         synchronized (root) {
+            ResourceGroupRuntimeInfo resourceGroupRuntimeInfo = resourceGroupRuntimeInfos.get(getId());
+            if (resourceGroupRuntimeInfo != null) {
+                return descendantQueuedQueries + queuedQueries.size() + resourceGroupRuntimeInfo.getQueuedQueries() < maxQueuedQueries;
```

Comment:
so `resourceGroupRuntimeInfo.getQueuedQueries()` returns "number of queued queries in the resource group EXCEPT this coordinator"?

Reply:
Yes, exactly.